include/highfive/H5Exception.hpp

```diff
@@ -14,6 +14,8 @@
 
 #include "bits/H5Utils.hpp"
 
+#include <H5Ipublic.h>  // for hid_t
```

Comment:
Why is the include relative here? IMO it should be in between " " ?

Reply:
this is a header from the hdf5 lib... I will change the order